protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.12.0
+
+* Breaking change: changed PbMap to not create a map entry BuilderInfo object for each PbMap instance, instead it
```

Comment:
In the breaking change section you should describe the updates to the public api.
Outside that section you can describe the bug fixed.

That way it will be easier for the user to see if/what changes are needed to update.